addons/mrp/models/mrp_bom.py

```diff
@@ -145,6 +145,15 @@ def unlink(self):
             raise UserError(_('You can not delete a Bill of Material with running manufacturing orders.\nPlease close or cancel it first.'))
         return super(MrpBom, self).unlink()
 
+    @api.model
+    def _name_search(self, name='', args=None, operator='ilike', limit=100, name_get_uid=None):
+        args = args or []
+        if operator == 'ilike' and not (name or '').strip():
```

Comment:
What about '=', '=ilike', 'like', '=like' and all the opposite (not ilike...)

Reply:
@amoyaux fixed it